app/views/bulk_reference.scala.html

```diff
@@ -29,30 +30,33 @@
 
     @views.html.helpers.errorSummary(bulkReferenceForm)
 
-        <h1>@Messages("gmp.bulk_reference.header")</h1>
+        <h1 class="govuk-heading-l">@Messages("gmp.bulk_reference.header")</h1>
 
         @viewHelpers.form(action = routes.BulkReferenceController.post) {
 
-          <div class="form-group">
-              <div class="form-field">
+          <div class="govuk-form-group">
                 @gmpInput(bulkReferenceForm("email"),
```

Comment:
I thought the goal of the UI play frontend was to get away from using specifically designed components such as radios, checkboxes, etc.  Should this be using govukInput and no longer using @gmpInput?
If no, go ahead and close/resolve conversation.

Reply:
The changes I have done is only the part of the journey mentioned 
 in the ticket. This page may come in another ticket  and will be handled then